examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -328,7 +328,7 @@ void MergeBasis(const int dimFOM, const int nparam, const int max_num_snapshots,
     generator.endSamples(); // save the merged basis file
 
     int cutoff = 0;
-    generator.finalSummary(1e-7, cutoff, "mergedSV_" + name + ".txt");
+    generator.finalSummary(1e-7, cutoff, "mergedSV_" + name + ".txt",0,squareSV);
```

Comment:
It is irrelevant to the functionality, but adding space after the commas make it consistent.